Add "GDC 2025 Retrospective" article (#1047)
godotengine/godot-website: collections/_article/gdc-2025-retrospective.md

---
title: "GDC 2025: Retrospective"
excerpt: "Even if the team sent to the Game Developers Conference (GDC) was smaller in size this year, a lot happened in San Francisco. So here's a small recap!
"
categories: ["events"]
author: Adam Scott
image: /storage/blog/covers/gdc-2025-retrospective.webp
date: 2025-03-27 13:00:00
---

Even if the team sent to the [Game Developers Conference](https://gdconf.com/) (GDC) was smaller in size this year, a lot happened in San Francisco. So here's a small recap!

## Public talk

The Executive Director of the [Godot Foundation](https://godot.foundation), Emilio Coppola, gave [a talk](https://schedule.gdconf.com/session/building-godot-the-process-behind-the-engine-presented-by-w4-games/911990) at the GDC Partner Stage titled "Building Godot: The Process Behind the Engine".

<img src="/storage/blog/gdc-2025/godot-talk.webp" alt="Emilio Coppola, executive director of the Godot Foundation, is at a podium giving his talk at the GDC." />

> Emilio explained in detail how the project works internally, and how the Foundation actively work for the benefit of everyone involved.

Reviewer: ```suggestion
Emilio explained in detail how the project works internally, and how the Foundation actively works for the benefit of everyone involved.
```

The seats were almost full, but you'll have to trust me on this as I forgot to take a picture of the attendance. Though, I'm able to corroborate my claim with how many people were in line to have the chance to talk with Emilio. See it for yourself.

<img src="/storage/blog/gdc-2025/godot-talk-after.webp" alt="Emilio Coppola answering questions from multiple talk attendees." />

All of this wouldn't have been possible without the sponsor of the talk, [W4 Games](https://www.w4games.com/). Our many thanks to them.

## Godot meetup

Inspired by the successful [Blender meetup](https://bsky.app/profile/blender.org/post/3lk7d6lbhf22o) at the beginning of the week, we decided to try one on our own. [With only 2 days of notice](https://bsky.app/profile/adamscott.studio/post/3lkopw475z22y), we weren't expecting much.

<img src="/storage/blog/gdc-2025/godot-meetup.webp" alt="Group picture of the Godot meetup participants at the Yerba Buena Gardens in San Francisco during the GDC" />

But the Godot community shared the word and answered! We were stunned by the number of people that showed up.

> Those who attended the meetup had the opportunity to talk with regular Godot contributors, Emilio Coppola, and Ariel Manzur (co-founder of the Godot Engine and board member of the [Godot Foundation](https://godot.foundation)).

Reviewer: ```suggestion
Those who attended the meetup had the opportunity to talk with regular Godot contributors, Emilio Coppola, and Ariel Manzur (co-founder of the Godot Engine and board member of the Godot Foundation).
```
I don't think you need to link this a second time.

> Our many thanks to all of those who showed up! It was a pleasure to meet all of you. Fun fact: even if I called the end of the meetup at the 2-hour mark, somebody told me that the last people left nearly one hour after.

Reviewer: ```suggestion
Our many thanks to all of those who showed up! It was a pleasure to meet all of you. Fun fact: even if I called the end of the meetup at the two hour mark, somebody told me that the last people left nearly one hour after.
```

## Showcased Godot games
### Kneedle Drop: An Into the Woolderness Game

"A RHYTHM KNITTING GAME?!" That's how [Immortal Funk Games](https://www.immortalfunkgames.com/) describes their spin-off game taylor-made (and chosen!) for the [alt.ctrl.GDC showcase of alternative controllers](https://gdconf.com/alt-ctrl-gdc).

<img src="/storage/blog/gdc-2025/kneedle-drop-gameplay.webp" alt="Kneedle Drop gameplay, where two people hold each two controllers shaped as knitting needles." />

> In this game that mimic the essence of knitting as well as Guitar Hero does for playing guitar, either you tap the point of the needles, or you need to tap the left controller on the right one from one side or the other, all while keeping the rhythm.

Reviewer: ```suggestion
In this game, that mimics the essence of knitting as well as Guitar Hero does for playing guitar, either you tap the point of the needles, or you tap the left controller on the right one from one side or the other, all while keeping the rhythm.
```

Author: I'm not sure about the first comma, though. It's valid with a comma there, but I doesn't have the rhythm I want to give to the sentence.

Reviewer: Yeah, it's fine without the first comma :). I still think it's better (less clunky) without the "need to" though.

Author: > Yeah, it's fine without the first comma :). I still think it's better (less clunky) without the "need to" though.

Oh, completely didn't see the "need to" suggestion. I'll remove it ASAP!

<img src="/storage/blog/gdc-2025/kneedle-drop-team.webp" alt="The team behind Kneedle Drop, in front of their game showing on a screen." />

The team was explaining to me how nice working with Godot is, especially for artists.

Be sure to check out [Into the Woolderness](https://www.immortalfunkgames.com/into-the-woolderness), the main project of the team. It's a cozy game about crafting and knitting garments for your dogs. Be reassured (or sad), as this one will play on a standard controller.

### Year Unknown

> [Year Unknown](https://store.steampowered.com/app/2565520/Year_Unknown/) is a game released in October last year by Julian Heuser. It is a "walking simulator" about exploring a minimalist landscape with the help of two AIs, trying to uncover secrets about humanity.

Reviewer: The Steam link presumably needs that curator thingy added.

Author: It already has.

Reviewer: I don't really understand it, but I'm talking about the stuff added in #1013 with ?curator_clanid=41324400 at the end of each Steam link.

---
title: "GDC 2025: Retrospective"
excerpt: "Even if the team sent to the Game Developers Conference (GDC) was smaller in size this year, a lot happened in San Francisco. So here's a small recap!
"
categories: ["events"]
author: Adam Scott
image: /storage/blog/covers/gdc-2025-retrospective.webp
date: 2025-03-27 13:00:00
---

Even if the team sent to the [Game Developers Conference](https://gdconf.com/) (GDC) was smaller in size this year, a lot happened in San Francisco. So here's a small recap!

## Small team perhaps, but as active as ever

> Last year, there was the (humongous) Godot/W4 Games Pavilion, the Foundation invited a lot of contributors to come in order to have enough qualified people to talk about the engine. As this year there was no engine booth on the GDC expo floor at all, there wasn't a need for a delegation of contributors.

Reviewer: ```suggestion
Last year, there was the (humongous) Godot/W4 Games Pavilion, and the Foundation invited a lot of contributors to come in order to have enough qualified people to talk about the engine. As this year there was no engine booth on the GDC expo floor at all, there wasn't a need for a delegation of contributors.
```
Needs the "and" (or similar, like "so").

> Last year, there was the (humongous) Godot/W4 Games Pavillon, the Foundation invited a lot of contributors to come in order to have enough qualified people to talk about the engine. As this year there was no engine booths at the GDC expo floor at all, the need for a delegation of contributors wasn't there.

Reviewer: ```suggestion
Last year, there was the (humongous) Godot/W4 Games Pavilion, and the Foundation invited a lot of contributors to come in order to have enough qualified people to talk about the engine. As this year there was no engine booth on the GDC expo floor at all, there wasn't a need for a delegation of contributors.
```

> The Foundation sent instead a small team composed of two board members of the [Godot Foundation](https://godot.foundation), HP van Braam and Clay John, and its Executive Director, Emilio Coppola. They were busy all week, going meeting to meeting (and sometimes even corporate parties), representing the interests of the engine with current and future sponsors or partners.

Reviewer: ```suggestion
The Foundation instead sent a small team composed of two board members of the [Godot Foundation](https://godot.foundation), HP van Braam and Clay John, and its Executive Director, Emilio Coppola. They were busy all week, going from meeting to meeting (and sometimes even corporate parties), representing the interests of the engine with current and future sponsors or partners.
```

## Public talk

The Executive Director of the Godot Foundation, Emilio Coppola, gave [a talk](https://schedule.gdconf.com/session/building-godot-the-process-behind-the-engine-presented-by-w4-games/911990) at the GDC Partner Stage titled "Building Godot: The Process Behind the Engine".

<img src="/storage/blog/gdc-2025/godot-talk.webp" alt="Emilio Coppola, executive director of the Godot Foundation, is at a podium giving his talk at the GDC." />

Emilio explained in detail how the project works internally, and how the Foundation actively works for the benefit of everyone involved.

The seats were almost full, but you'll have to trust me on this as I forgot to take a picture of the attendance. Though, I'm able to corroborate my claim with how many people were in line to have the chance to talk with Emilio. See it for yourself.

<img src="/storage/blog/gdc-2025/godot-talk-after.webp" alt="Emilio Coppola answering questions from multiple talk attendees." />

All of this wouldn't have been possible without the sponsor of the talk, [W4 Games](https://www.w4games.com/). Our many thanks to them.

## Godot meetup

> Inspired by the successful [Blender meetup](https://bsky.app/profile/blender.org/post/3lk7d6lbhf22o) at the beginning of the week, we decided to try one on our own. [With only 2 days of notice](https://bsky.app/profile/adamscott.studio/post/3lkopw475z22y), we weren't expecting much.

Reviewer: ```suggestion
Inspired by the successful [Blender meetup](https://bsky.app/profile/blender.org/post/3lk7d6lbhf22o) at the beginning of the week, we decided to try one of our own. [With only 2 days of notice](https://bsky.app/profile/adamscott.studio/post/3lkopw475z22y), we weren't expecting much.
```

<img src="/storage/blog/gdc-2025/godot-meetup.webp" alt="Group picture of the Godot meetup participants at the Yerba Buena Gardens in San Francisco during the GDC" />

But the Godot community shared the word and answered! We were stunned by the number of people that showed up.

Those who attended the meetup had the opportunity to talk with regular Godot contributors, Emilio Coppola, and Ariel Manzur (co-founder of the Godot Engine and board member of the Godot Foundation).

> Our many thanks to all of those who showed up! It was a pleasure to meet all of you. Fun fact: even if I called the end of the meetup at the two hour mark, somebody told me that the last people left nearly one hour after.

Reviewer: ```suggestion
Many thanks to all of those who showed up! It was a pleasure to meet all of you. Fun fact: even if I called the end of the meetup at the two hour mark, somebody told me that the last people left nearly one hour after.
```
"Our thanks" or "Many thanks" is OK - for some reason "Our many thanks" isn't!

The team was explaining to me how nice working with Godot is, especially for artists.

Be sure to check out [Into the Woolderness](https://www.immortalfunkgames.com/into-the-woolderness), the main project of the team. It's a cozy game about crafting and knitting garments for your dogs. Be reassured (or sad), as this one will play on a standard controller.

### Year Unknown

[Year Unknown](https://store.steampowered.com/app/2565520/Year_Unknown/) is a game released in October last year by Julian Heuser. It is a "walking simulator" about exploring a minimalist landscape with the help of two AIs, trying to uncover secrets about humanity.

<img src="/storage/blog/gdc-2025/year-unknown.webp" alt="Julian Heuser before someone playing his game, Year Unknown." />

The game was showcased at the [IGF](https://igf.com/) Pavillon as [a finalist for "Best Student Game" for the "2025 Independent Games Festival"](https://igf.com/finalists-and-winners). Even if the game didn't ultimately win the prize, few developers can actually say that they were nominated by the IGF. Congratulations, Julian!

## Conclusion

> You know, as Godot is an international project and as its contributors usually work remotely, events like this one are usually the only way for us to actually meet. Meeting numerous colleagues, great corporate partners, and people that are so passionate about the project made the trip worthwhile.

Reviewer: ```suggestion
As Godot is an international project and its contributors mostly work remotely, events like this one are usually the only way for us to actually meet. Meeting numerous colleagues, great corporate partners, and people that are so passionate about the project made the trip worthwhile.
```
Changed to "mostly" to avoid close repetition of "usually".

> I'm considering myself lucky to be able to travel to San Francisco representing a project that I deeply care of. I returned home happy and fulfilled — and with a flu.

Reviewer: ```suggestion
I'm considering myself lucky to be able to travel to San Francisco representing a project that I deeply care for. I returned home happy and fulfilled — and with the flu.
```
(or "with a cold", but not "a flu").

Author: Thinking about it, it's mostly a cold, not the flu. Thanks!

Reviewer: Get well soon :)
